packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -31,14 +31,19 @@ import * as Classes from "../../common/classes";
 
 export interface IOmniboxProps<T> extends IListItemsProps<T> {
     /**
+     * React child to render when query is empty.
+     */
+    initialRenderer?: JSX.Element;
+
+    /**
      * Custom renderer for an item in the dropdown list. Receives a boolean indicating whether
      * this item is active (selected by keyboard arrows) and an `onClick` event handler that
      * should be attached to the returned element.
      */
     itemRenderer: (itemProps: ISelectItemRendererProps<T>) => JSX.Element;
 
     /** React child to render when filtering items returns zero results. */
-    noResults?: string | JSX.Element;
```

Comment:
why remove this? lets just use `React.ReactChild` in both places (this and new `initialRenderer`).